### ISSUE TITLE
conventionalcommits types use same heading level as new version

### ISSUE THREAD

**Issue by tilowestermann** [release-it/conventional-changelog#54]
When generating the changelog with the `conventionalcommits` preset, the generated changelog will look like this:

```
### [1.0.1](<Url to diff>) (2022-09-06)


### Bug Fixes

* <commit message> ([<commit id>](<link to commit>))
```

I would expect the heading level of the new version to be 2 (`## [1.0.1] ...`)

**Comment by webpro** [comment]
Thanks for raising this, @tilowestermann. I now understand better what's happening here. This release-it plugin does not touch the header (level). It is now clear to me that conventional-changelog uses the **first** `#` heading level for minor and major releases, and the **second** `##` level for patch releases. It isn't very obvious, but the examples show this at https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-writer/README.md

This is now reflected in the tests here: https://github.com/release-it/conventional-changelog/commit/9aecd6a216578a2f4767005ecd1ae80e64e1d5c8

However, I fail to understand why you are seeing the **third** `###` level in the header. I also can't find settings to influence this, except for using custom hbs templates. Do you have a reproducible test case or something?

**Comment by tilowestermann** [comment]
Hey @webpro , thanks for your reply. I've set up a repository demonstrating the issue: https://github.com/tilowestermann/release-it-conventional-changelog/commits/main

By doing this, I noticed that the first release (1.1.0) including a single feature commit set the levels correctly, while the second one (1.1.0) including a single fix commit does not

**Comment by webpro** [comment]
Please understand release-it doesn't touch these results. They're generated by `conventional-changelog`, the dependency that powers this plugin. And maybe it's the `conventionalcommits` preset that handles things differently, as this plugin uses `angular` in the tests (which seems to behave as I described).

**Comment by matatk** [comment]
[`conventional-changelog-conventionalcommits` v5.0.0 fixed this problem](https://github.com/conventional-changelog/conventional-changelog/releases/tag/conventional-changelog-conventionalcommits-v5.0.0), but there has not yet been a release of `conventional-changelog` that depends on `conventional-changelog-conventionalcommits` v5.0.0, which this plugin could use. I filed an [issue to request a new release of conventional-changelog](https://github.com/conventional-changelog/conventional-changelog/issues/940).

In the meantime, I found that it's [possible to use `overrides` in `package.json` to force this new version to be used](https://stackoverflow.com/a/70396201/1485308). I've _just_ tried it out, and it seems to be working so far.
